### PR TITLE
updated and organized translations

### PR DESCRIPTION
**Pull request by jonathanraes**

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -31,21 +45,36 @@
     <string name="storage_permission_explanation">Geef alstublieft toegang tot de opslag om blockchain te kunnen gebruiken.</string>
 
     <!--NFC Scanning-->
+    <string name="passportconnection">Verbind met Reisdocument</string>
     <string name="nfc_not_supported_error">Dit apparaat ondersteunt geen NFC.</string>
-    <string name="nfc_disabled_error">NFC is uitgeschakeld, zet dit aan en houd het reisdocument tegen de achterzijde van dit apparaat om te scannen.</string>
+    <string name="nfc_disabled_error_snackbar">NFC is uitgeschakeld, zet dit aan en houd het reisdocument tegen de achterzijde van dit apparaat om te scannen.</string>
+    <string name="nfc_disabled_snackbar_action">Zet NFC aan</string>
     <string name="nfc_enabled">Houd het reisdocument tegen de achterzijde van dit apparaat.</string>
+    <string name="general_error">Verbinden mislukt, check a.u.b. of het document compatibel is.</string>
```
Comment (wkmeijer) [inline app/src/main/res/values-nl/strings.xml]: check = controleer

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -31,21 +45,36 @@
...
+
     <string name="NFC_error">Kan NFC-scan niet afronden, probeer opnieuw.</string>
+    <string name="start_con_button">Start NFC connectie</string>
+    <string name="scan_doc_details">Scan eerst de code op het document</string>
+
+    <!--Transaction History-->
     <string name="transaction_history">Transactiegeschiedenis</string>
+    <string name="no_transactions_found">No transaction history found</string>
 
+    <string name="wrong_document_details">Document details zijn onjuist ingevoerd</string>
```
Comment (wkmeijer) [inline app/src/main/res/values-nl/strings.xml]: 'document gegevens' lijkt me hier beter

File: app/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -31,21 +45,36 @@
...
+    <string name="edit_election">Verkiezing aanpassen</string>
 
+    <!--Result Activity-->
     <string name="proceed_home">Volgende kiezer</string>
+    <string name="waiting_confirmation">Wachten op bevestiging</string>
     <string name="proceed_cast_vote">Stempas verzilveren</string>
     <string name="cancel_vote">Annuleren</string>
-
     <string name="authorization_failed">Geen stempassen gevonden.</string>
+    <string name="authorization_succesful">Authorizatie geslaagd, gelieve bevestigen.</string>
```
Comment (wkmeijer) [inline app/src/main/res/values-nl/strings.xml]: authorisatie

File: app/src/main/res/values-nl/strings.xml [hunk 4]
```diff
@@ -69,37 +104,22 @@
...
-
     <string-array name="init_array">
         <item>Initialiseren</item>
         <item>\u00A0Initialiseren.</item>
         <item>\u00A0\u00A0Initialiseren..</item>
         <item>\u00A0\u00A0\u00A0Initialiseren...</item>
     </string-array>
+
+    <!--Transaction History-->
+    <string name="transaction_received_item_format_title">%1$d stempas ontvagen \n%2$s %3$s</string>
```
Comment (wkmeijer) [inline app/src/main/res/values-nl/strings.xml]: ontvagen = ontvangen

**Review by wkmeijer — CHANGES_REQUESTED** [review]
Fixes the code lint language warnings. I have some minor requests.

File: app/src/main/res/values-nl/strings.xml [hunk 5]
```diff
@@ -31,21 +45,36 @@
     <string name="storage_permission_explanation">Geef alstublieft toegang tot de opslag om blockchain te kunnen gebruiken.</string>
 
     <!--NFC Scanning-->
+    <string name="passportconnection">Verbind met Reisdocument</string>
```
Comment (klikooo) [inline app/src/main/res/values-nl/strings.xml]: Reisdocument without capital

**Review by wkmeijer — APPROVED** [review]

**Review by klikooo — APPROVED** [review]
Good job

**Pull request merged**